db/puppies.js

```diff
@@ -0,0 +1,99 @@
+const client = require("./client");
+
+async function createPuppy({
+    name,
+    description,
+    breed,
+    weight,
+    size,
+    pedigree,
+    isVaccinated,
+    isNeutered,
+    gender,
+    isAvailable,
+    price
+}) {
+    try{
+        const { rows: [ puppy ] } = await client.query(`
+            INSERT INTO puppies(name, description, breed, weight, size, pedigree, isVaccinated, isNeutered, gender, isAvailable, price)
+            VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11)
```

Comment:
Think we need "ON CONFLICT (name) DO NOTHING" on the line under here.

Reply:
Done